hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/dao/LabelResponseConverter.java

```diff
@@ -25,18 +25,40 @@
   Single<List<Label>> convertToLabelList(GetLabelsResponse response) {
     return Single.just(
         response.getLabelsList().stream()
-            .map(label -> new DefaultLabel(label.getId(), label.getData().getKey()))
+            .map(
+                label ->
```

Comment:
the construction of the label is the same in each case, from a proto Label to a gql Label, can dedupe the code.